### PR TITLE
0.15.9

### PR DESCRIPTION
**Pull request by audrow**
Bumps Humble from 0.15.8 to 0.15.9.

### CODE REVIEW

**Review by clalancette — APPROVED** [review]
Seems reasonable to me, but let's wait for @MichaelOrlov to approve before merging and tagging.

File: rosbag2_transport/CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,40 @@
...
+  - Use +0.1 and -0.1 in decrease/increase rate formula instead of
+  multiply by factor of the 1.1 and 0.9 respectively.
+  (cherry picked from commit 95f78b63f8cebc8c4a902b4783e4e7a1d0271d02)
+  # Conflicts:
+  #	rosbag2_transport/src/rosbag2_transport/player.cpp
+  * Address merge conflicts after auto-backporting
+  ---------
+  Co-authored-by: Michael Orlov <michael.orlov@apex.ai>
+* Contributors: mergify[bot]
+
```
Comment (MichaelOrlov) [inline rosbag2_transport/CHANGELOG.rst]:
```suggestion
* [humble] Give proper log message for `--start-paused` (backport `#1537 <https://github.com/ros2/rosbag2/issues/1537>`_) (`#1541 <https://github.com/ros2/rosbag2/issues/1541>`_)
* [humble] `Recording stopped` prints only once. (backport `#1530 <https://github.com/ros2/rosbag2/issues/1530>`_) (`#1535 <https://github.com/ros2/rosbag2/issues/1535>`_)
* [humble] Bugfix for incorrect playback rate changes when pressing buttons (backport `#1513 <https://github.com/ros2/rosbag2/issues/1513>`_) (`#1515 <https://github.com/ros2/rosbag2/issues/1515>`_)
* Contributors: Michael Orlov <michael.orlov@apex.ai>, Christoph Froehlich <christoph.froehlich@ait.ac.at>, Tomoya Fujita <Tomoya.Fujita@sony.com>
```

**Review by MichaelOrlov — CHANGES_REQUESTED** [review]
@audrow Looks good among one section in `rosbag2_transport` package which has quirks from merge conflicts.  
I've made proposal for fix in the comment.

**Comment by audrow** [comment]
I've applied the suggestion. @MichaelOrlov, let me know if everything looks good to you.

**Review by MichaelOrlov — APPROVED** [review]
LGTM.

**Pull request merged**